tools/pot/openvino/tools/pot/data_loaders/image_loader.py

```diff
@@ -36,5 +38,24 @@ def _read_and_preproc_image(self, img_path):
 
         if image is None:
             raise Exception('Can not read the image: {}'.format(img_path))
-
-        return prepare_image(image, self.shape[-2:], self._crop_central_fraction)
+        print(image.shape)
+        return prepare_image(image, self._layout, self.shape[-2:], self._crop_central_fraction)
+
+    def get_layout(self, input):
```

Comment:
We should determine the layout based on the shape of the model input **nodes** but not on the input data.